src/health/HealthController.ts

```diff
@@ -0,0 +1,46 @@
+import {Controller, Get} from '@nestjs/common';
+import {
+    HealthCheck,
+    HealthCheckResult,
+    HealthCheckService,
+    HttpHealthIndicator,
+    MicroserviceHealthIndicator
+} from '@nestjs/terminus';
+import config from '../config';
+import {MicroserviceOptions, RedisOptions, Transport} from '@nestjs/microservices';
+
+@Controller('health')
+export class HealthController {
+    constructor(
+        private readonly health: HealthCheckService,
+        private readonly http: HttpHealthIndicator,
+        private readonly microservice: MicroserviceHealthIndicator,
+    ) {
+    }
+
+    @Get()
+    @HealthCheck()
+    public check(): Promise<HealthCheckResult> {
+        return this.health.check([
+            () => this.http.pingCheck('api', config.internalApi.host, {
+                headers: {
+                    'Content-Type': 'application/json',
+                },
+                auth: {
+                    username: config.internalApi.username,
+                    password: config.internalApi.password,
+                },
```

Comment:
This isn't really required for the healthcheck, also the ping check should ping the endpoint `/actives` as that will also test if the API is responding correctly by returning a list of active alerts.